lib-rs/src/lib.rs

```diff
@@ -6,6 +6,67 @@
 //! * Convenience wrapper and builder for C strings.
 //! * Macro for creating owning data types.
 //! * Macro for creating iterators from access by index and length.
+//!
+//! # Example usage
```

Comment:
Is that something people do? I was annoyed by the duplication, so I would not mind doing this.

Reply:
Yep, at least in some projects I've seen, although you have to be careful that the README still looks OK, but since in your case you're copying verbatim, you might as well.